### DISCUSSION TITLE
Looker Studio integration issue

### DISCUSSION THREAD

**Discussion by AlexShku** [outline/outline#6398] [Self Hosting]
Hello! 
I'm trying to insert link to Google Looker Studio but nothing happens. 
Any buttons doesn't work. 

Maybe someone knows how to resolve this issue. 
![image](https://github.com/outline/outline/assets/91185761/7df92182-bc74-4602-bcfd-0ebad9ee36b7)

`Outline Version - v0.74.0 (Self Hosted)`

**Comment by tommoor** [comment]
It seems like the format does not match expected, try removing `/preview` from the end? If that doesn't work please post the full link – you can change individual characters to obfuscate

**Reply by AlexShku** [reply]
Hello, Tom! 
Here the test link - https://lookerstudio.google.com/reporting/0B_U5RNpwhcE6QXg4SXFBVGUwMjg/page/6zXD
Nothing works for me. 
I tried different options.

**Reply by tommoor** [reply]
In production this link seems to work fine, so perhaps it was fixed already on `main`  🤔 

<img width="803" alt="image" src="https://github.com/outline/outline/assets/380914/0e0520fe-fe59-4f6f-b7a0-09340abac68e">

**Reply by AlexShku** [reply]
Hello! I tried recreate everything from the scratch but I have the same issue. 
Here is my envs. Maybe it something connected with some variables. 

```
#RDS
db_instance_family 		      = "postgres12"
db_instance_engine_version 	  = "12.16"

#REDIS
redis_engine_version       = "7.0"

#WIKI OUTLINE VARS
node_env     = "production"

#File  Storage
file_storage                 = "s3"
file_storage_upload_max_size = "512000000"
aws_s3_force_path_style = false

#App Settings
default_language = "en_US"
force_https      = true
enable_updates   = true

#Tunables
web_concurrency               = "1"
log_level                     = "verbose"
rate_limiter_enabled          = true
rate_limiter_requests         = "1000"
rate_limiter_duration_window  = "60"
maximum_import_size           = "70000000"
development_unsafe_inline_csp = false
```

**Reply by AlexShku** [reply]
Sorry! The problem has been solved. 
The issue: wrong link format.
